LibrariesPrototype/Screens/NativeAPI/HeroDetail/ViewControllers/HeroDetailViewController.swift

```diff
@@ -74,17 +70,27 @@ class HeroDetailViewController: LTViewController {
         view.backgroundColor = .green
         return view
     }()
+    
+    static let cardCornerRadius: CGFloat = 5
 
     // MARK: - Variables
     private var viewModel: HeroDetailViewModel!
+    public var heroId: Int? {
```

Comment:
Could be something like this ?
var heroId: Int! {
        didSet {
            viewModel.setHeroId(heroId)
        }
    }